client/src/pages/BidderSignUp.js

```diff
@@ -0,0 +1,183 @@
+/* eslint-disable no-trailing-spaces */
```

Comment:
You have duplicated code in this PR and in https://github.com/LauraSantiag0/Love-Me-Tender/pull/30/files . This duplicates reviewer's work. Please remove not relevant files from this PR